tensorflow/examples/label_image/BUILD

```diff
@@ -51,6 +51,16 @@ tf_cc_binary(
     }),
 )
 
+py_binary(
+    name = "label_image_py",
+    main = "label_image.py",
```

Comment:
main should come after srcs.

Reply:
Ahh interesting. @freedomtan can you change that.

Reply:
@sb2nov done.